### ISSUE TITLE
Thanks to the author for the excellent work, but I wonder why group_centriod = torch.zeros_like (sorted_group_xyz), why is the value of group_centriod all 0?

### ISSUE THREAD

**Issue by dengyingxu** [hancyran/RepSurf#5]
https://github.com/hancyran/RepSurf/blob/ef873f8f36aa224e4b08edaab21330a828dac8f9/modules/repsurface_utils.py#L129

Thanks to the author for the excellent work, but I wonder why group_centriod = torch.zeros_like (sorted_group_xyz), why is the value of group_centriod all 0?

**Comment by hancyran** [comment]
We use the coordinates of one centroid and its two/K neighbors to compute the normal vectors, and we normalize its neighbors' cooridnates as relative position, so the centroid should be (0, 0, 0)
